front/app/components/admin/Graphs/MultiBarChart/index.tsx

```diff
@@ -99,6 +100,14 @@ const MultiBarChart = <Row,>({
         );
     };
 
+  const tickFormatter = (value: string) => {
+    const limit = 42; // maximum characters before ellipses
+
+    if (value.length < limit) return value;
+
+    return truncate(value, limit);
```

Comment:
```suggestion
    return truncate(value, 42);
```

truncate already includes this behavior. If str is longer than limit, it returns the truncated string, otherwise it returns value.